src/main/java/dev/binclub/javaception/klass/Klass.java

```diff
@@ -71,4 +101,27 @@ public MethodInfo findMethod(String name, Type[] descriptor) {
 		}
 		return null;
 	}
+	
+	public int getMethodID(String name, String descriptor) {
+		for (int i = 0; i < methods.length; i++) {
+			MethodInfo method = methods[i];
+			if (name.equals(method.name) && method.signature.equals(descriptor)) {
```

Comment:
I think you meant `method.descriptor.equals(descriptor)`

Reply:
don't want to compare arrays